scripts/help-text/bump-changelog.txt

```diff
@@ -0,0 +1,9 @@
+Usage: ./scripts/bump-changelog [OPTION]...
+Moves everything under the "Unreleased" heading of the changelog to a new version heading based on the version number
```

Comment:
Does this mean user should bump the version in gradle.properties before running this?  If so, document that.

Reply:
Yes, and I'll make sure this is explicitly stated. I'll also explicitly state this in the dev guide.

As for the naming, I chose `bump-changelog` because it should only be used when bumping to a new version, however `update-changelog` would make more sense to whoever is using it. I'll change it, and I'll also ensure that if someone tries to run it when not bumping the project version, it'll explain to them that the script is only needed when bumping the project version.